### DISCUSSION TITLE
how to know if application has gotten linked to zluda

### DISCUSSION THREAD

**Discussion by brcisna** [lshqqytiger/ZLUDA#47] [Q&A]
Hello All,

Debian 13 ( testing) 
Completely new to Zluda.

Am trying to see if i can get wunjo ai V2 to start and run with Zluda

What are the tell tale signs that 'whatever application' is actually utilizing ZLUDA.

Excuse me for my ignorance. I have tried reading all the ZLUDA documenation,,and can get wunjo to start with 
LIB_PATH,/path/to/zluda/ ,,etc breifcase dev.

wunjo launches as normal,,and in console i never see any indications of ZLUDA,,,output?

TIA

**Comment by lshqqytiger** [comment]
The application (wunjo ai) seems to use PyTorch internally. You can follow [this](https://github.com/lshqqytiger/ZLUDA?tab=readme-ov-file#pytorch), but please consider using official release of PyTorch built with ROCm. (it will be better than ZLUDA in most cases)

**Reply by brcisna** [reply]
@lshqqytiger 

Thank You very much!
Am very green to all the various backends for doing AI tasks.  wunjo ai,,is a very cool app,,,,,especially for a beginner like me...
I just wanted to see if i could get ZLUDA to work,,as a challenge,,to see if it was snappier ,but will call doing pytorch with rocm good.

Thanks again
